src/ecalc/libraries/libecalc/common/libecalc/core/models/compressor/base.py

```diff
@@ -98,6 +100,10 @@ def _validate_operational_conditions(
             None,
         ]
 
+        # calculations are performed only if rate is positive.
+        # do not check validity of operational conditions if rate is zero
+        check_validity_for_time_step = np.any(rate > 0, axis=0) if np.ndim(rate) == 2 else rate > 0
```

Comment:
denne vil returnere hva? enkel bool enner ndarray? ndarray med indekser  for hvilke som evaluerer til henholdsvis true og false?